frontend/mutators.ts

```diff
@@ -47,6 +50,8 @@ export const mutators = {
   },
   deleteIssues: async (tx: WriteTransaction, ids: string[]): Promise<void> => {
     for (const id of ids) {
+      await deleteIssueComments(tx, id);
```

Comment:
Was this just wrong before?

Reply:
ya, this function actually wasn't being used and was implemented incorrectly.